### ISSUE TITLE
turn verbose for ensemble

### ISSUE THREAD

**Issue by hainm** [Amber-MD/cpptraj#85]
I am playing with remd stuff and CpptrajState.

For `trajin` keyword, I got not verbose (since pytraj turns it off)

``` python
state = pt.datafiles.load_cpptraj_state('''
        parm ala2.99sb.mbondi2.parm7
        trajin rem.nc.000
        rms
        ''')

state.run()
print(state.data)
```

--> output `<pytraj.datasets.DatasetList - 1 datasets>`

but when using, I got a bunch of lines:
Input code

``` python
state = pt.datafiles.load_cpptraj_state('''
        parm ala2.99sb.mbondi2.parm7
        ensemble rem.nc.000 remdtraj remdtrajtemp 300.
        rms
        ''')
state.run()
print(state.data)
```

output:

``` bash
TIME: Run Initialization took 0.0001 seconds.

BEGIN ENSEMBLE PROCESSING:
        ENSEMBLE: OPENING 4 REMD TRAJECTORIES
.....................................................
ACTION SETUP FOR PARM 'ala2.99sb.mbondi2.parm7' (1 actions):
  0: [rms]
        Target mask: [*](22)
        Reference mask: [*](22)
----- rem.nc.000 (1-10, 1) -----
 0% 11% 22% 33% 44% 56% 67% 78% 89% 100% Complete.

Read 10 frames and processed 10 frames.
TIME: Trajectory processing: 0.0008 s
TIME: Avg. throughput= 12626.2626 frames / second.

ENSEMBLE ACTION OUTPUT:

DATASETS:
  4 data sets:
        RMSD_00000%0 "RMSD_00000" (double, rms), size is 10
        RMSD_00001%1 "RMSD_00001%1" (double, rms), size is 10
        RMSD_00002%2 "RMSD_00002%2" (double, rms), size is 10
        RMSD_00003%3 "RMSD_00003%3" (double, rms), size is 10
---------- RUN END ---------------------------------------------------
---------- RUN BEGIN -------------------------------------------------
Warning: No actions/output trajectories specified.

DATASETS:
  4 data sets:
        RMSD_00000%0 "RMSD_00000" (double, rms), size is 10
        RMSD_00001%1 "RMSD_00001%1" (double, rms), size is 10
        RMSD_00002%2 "RMSD_00002%2" (double, rms), size is 10
        RMSD_00003%3 "RMSD_00003%3" (double, rms), size is 10
---------- RUN END ---------------------------------------------------
<pytraj.datasets.DatasetList - 4 datasets>
```

**Comment by hainm** [comment]
so how can I turn the verbose off?

info
```bash
cpptraj version =  V16.00b
cpptraj internal version =  V4.2.5
cpptraj compiled flag =   -DHASBZ2 -DHASGZ -DBINTRAJ -D_OPENMP
```

**Comment by drroe** [comment]
Hm, this is because cpptraj calls SetWorldSilent() internally to prevent the Setup etc from each ensemble member to spit out output. Seems like SetWorldSilent will have to be more than just a toggle. I'll work on it.

**Comment by hainm** [comment]
thanks.

**Comment by drroe** [comment]
@hainm this should hopefully be addressed by #466. There is a function now called `SuppressAllOutput()` in `CpptrajStdio.h` that will turn off all STDOUT output for good. STDOUT output can now also be redirected to a separate file. Let me know if you have questions.

**Comment by hainm** [comment]
thx.
